### PR TITLE
Update docs for generating service account tokens.

### PR DESCRIPTION
**Pull request by upodroid**
https://github.com/hashicorp/vault-plugin-auth-gcp/pull/115

### CODE REVIEW

File: website/content/docs/auth/gcp.mdx [hunk 1]
```diff
@@ -333,6 +344,8 @@ Read more on the
 
 ### GCE
 
+You can autogenerate this token in Vault versions newer than 1.8.2.
```
Comment (aphorise) [inline website/content/docs/auth/gcp.mdx:347]: Vault 1.7.2? - or is this something intended for the future 1.8.2 release :smile:

Comment (upodroid) [reply website/content/docs/auth/gcp.mdx:347]: Should be the next release. Who willr review the patch in the plugin review?

File: website/content/docs/auth/gcp.mdx [hunk 2]
```diff
@@ -31,16 +31,23 @@ repository.
 ### Via the CLI Helper
 
 Vault includes a CLI helper that obtains a signed JWT locally and sends the
-request to Vault. This helper is only available for IAM-type roles.
+request to Vault.
 
 ```shell-session
+# This for authentication to vault outside of Google Cloud
```
Comment (taoism4504) [inline website/content/docs/auth/gcp.mdx:37]:
```suggestion
# This for authentication to Vault outside of Google Cloud
```

**Review by taoism4504 — COMMENTED** [review]
one nit, but otherwise looks good.

**Comment by upodroid** [comment]
Can you find an engineer to review https://github.com/hashicorp/vault-plugin-auth-gcp/pull/115? That needs to be merged before this one. 

I'll fix the nits.

**Comment by aphorise** [comment]
Closing in favour of #16787 - @upodroid FYI.

**Pull request closed without merging**